### PR TITLE
Builder commands cleanup

### PR DESCRIPTION
**Pull request by kthoms**
Changes done automatically by Buildship tooling:
- multiple org.eclipse.xtext.ui.shared.xtextBuilder commands reduced by
one
- added org.eclipse.buildship.core.gradleprojectbuilder

### CODE REVIEW

File: org.eclipse.xtext.core.idea.tests/.project [hunk 1]
```diff
@@ -11,17 +11,12 @@
 			</arguments>
 		</buildCommand>
 		<buildCommand>
-			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
```
Comment (cdietrich) [inline org.eclipse.xtext.core.idea.tests/.project]: why is this one removed?

File: org.eclipse.xtext.core.idea.tests/.project [hunk 2]
```diff
@@ -11,17 +11,12 @@
...
-			<arguments>
-			</arguments>
-		</buildCommand>
-		<buildCommand>
 			<name>org.eclipse.jdt.core.javabuilder</name>
 			<arguments>
 			</arguments>
 		</buildCommand>
 		<buildCommand>
-			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
```
Comment (cdietrich) [inline org.eclipse.xtext.core.idea.tests/.project]: same here

File: org.eclipse.xtext.core.idea.tests/.project (hunk 1 above)
Comment (cdietrich) [inline org.eclipse.xtext.core.idea.tests/.project]: for me this looks like there must be a misconfig somewhere in gradle

Comment (cdietrich) [inline org.eclipse.xtext.core.idea.tests/.project]: https://github.com/xtext/xtext-gradle-plugin/blob/9a0a1dafc415aacc62c763cee9c8bf82f9e285f9/xtext-gradle-plugin/src/main/java/org/xtext/gradle/XtextBuilderPlugin.xtend should add that

Comment (cdietrich) [inline org.eclipse.xtext.core.idea.tests/.project]: @oehme do you have an idea why this is not working

Comment (kthoms) [inline org.eclipse.xtext.core.idea.tests/.project]: I just re-imported the project with the gradle import wizard and it kept stable, i.e. removing the builder here. Don't now why.

Comment (oehme) [inline org.eclipse.xtext.core.idea.tests/.project]: @cdietrich the builder was there twice, it only removed the duplicate.

Comment (cdietrich) [inline org.eclipse.xtext.core.idea.tests/.project]: ahhh sry for that it was there even 3 times

Comment (oehme) [inline org.eclipse.xtext.core.idea.tests/.project]: Must have been a bug in previous versions.

**Pull request merged**